ci/Jenkinsfile.linux

```diff
@@ -10,7 +10,7 @@ pipeline {
     /* Necessary image with Ubuntu 20.04 for older Glibc. */
     docker {
       label 'linux'
-      image 'statusteam/nim-status-client-build:1.6.1-qt5.15.2'
+      image 'statusteam/nim-status-client-build:1.6.7-qt5.15.2'
```

Comment:
Why a jump of 6 versions?

Reply:
some of them were docker images with bugs, I had to push them so that I could test with CI.
I can delete the ones that don't work so that we may only bump 1 version instead of 6

Reply:
You could have pushed to the same one, you just had to make sure to prune old images from CI slaves.

Reply:
yes, key thing i missed was purging old docker image from ci-slaves.
I'll use `statusteam/nim-status-client-build:1.6.3-qt5.15.2` here jumping 2 versions instead of 1.
I believe i was using `statusteam/nim-status-client-build:1.6.2-qt5.15.2` for the `golang` bump.

Reply:
would be cool if you could delete other tags I pushed, I seem to not have permission in Docker Hub for it.